webauthn/webauthn.py

```diff
@@ -103,6 +105,13 @@ def __init__(self, challenge, rp_name, rp_id, user_id, username,
                              ', '.join(self._attestation_forms))
         self.attestation = attestation
 
+        if userVerification != None:
+            userVerification = str(userVerification).lower()
```

Comment:
I don't think we need this line if we make the other change I'm suggesting below.

Reply:
The use of `.lower()` make the api more flexible since we can use it with upper case parameters.

Reply:
👍Sounds good. You're right that it makes it more consistent, and we already do this for the `attestation` value.